index.js

```diff
@@ -218,6 +220,7 @@ Lexer.prototype = {
       var tok = this.tok('comment', captures[2]);
       tok.buffer = '-' != captures[1];
       this.pipeless = true;
+      this.interpolationAllowed = tok.buffer;
```

Comment:
I'd say yes, simply because there is no way to add interpolation in comments otherwise.